bin/client_build_xgo

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Usage example:
+#> bin/client_build_xgo <os>/<arch>..
+#
+# Cross compile (Unix):
+#> bin/client_build_xgo linux/amd64
+#
+# Cross compile (OSX + Windows):
+#> bin/client_build_xgo darwin/amd64 windows/amd64
+#
+# Check if program has dynamic libraries:
+#> brew install readelf
+#> readelf -d build/client/mysterium_client
+
+XGO_TARGETS=`IFS=','; echo "$*"`
+if [ -z "$XGO_TARGETS" ]; then
+    printf "\e[0;31m%s\e[0m\n" "Missing targets!"
+    exit 1
+fi
+
+source bin/helpers/functions.sh
+source bin/helpers/load_environment.sh
+DIR_BUILD="build/package"
+DIR_TEMP=`mktemp -d ${DIR_BUILD}/${tempname}.XXXXXX`
+
+xgo --go=1.9.2 \
```

Comment:
I am using `1.10` for some time already, did not encounter any problems.

Reply:
what is 'xgo', where should I get it? Should this be documented in INSTALL.md or in a script that uses it?

Reply:
>> I am using 1.10 for some time already, did not encounter any problems.
Not PR scope. Register ideas for to backlog (created MYST-562)

Reply:
> what is 'xgo', where should I get it? Should this be documented in INSTALL.md or in a script that uses it?

Added xgo as project development dependency